src/test/java/com/google/api/codegen/testdata/go_library.baseline

```diff
@@ -635,54 +635,54 @@ func (it *StringIterator) NextPageToken() string {
     return it.nextPageToken
 }
 ============== file: google.golang.org/cloud/library/generated/library/client_test.go ==============
-package library
+package library_test
 
 import (
     gax "github.com/googleapis/gax-golang"
     "golang.org/x/net/context"
     google_example_library_v1 "google.golang.org/cloud/library/generated/google/example/library/v1"
```

Comment:
Not an immediate concern, but when we figure out the import path, we should give different names to the packages. It would be unfortunate if you have two packages called "library" and you always need both of them.